### PR TITLE
Confluence README Page update

### PR DESCRIPTION
**Pull request by imkrishnasarathi**
#### Summary
This pull request updates the README for the Confluence repository to have a Table of Contents and split the Admin Guide to another file in `/docs/` folder.

#### Ticket Link

This pull request fixes this issue - https://github.com/mattermost/mattermost/issues/25174

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,92 +1,35 @@
...
+
+- [Overview](#overview)
+- [Features](#features)
+- [Admin Guide](/docs/admin-guide.md)
+- Contribute
+  - [Development](#development)
+  - [Maintainers](#maintainers)
+  - [Platform and Tools](#platform-and-tools)
+  - [Set up CircleCI](#set-up-circleci)
+- [License](/LICENSE)
```
Comment (mickmister) [inline README.md:11]: I think we can remove the following sections from the document, as they are either outdated or contain irrelevant information:

- Maintainers
- Platform and Tools
- Set up CircleCI

In the "Development" section, we can just have this sentence:

```
Read the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) documentation for more information about developing and extending plugins.
```

File: README.md [hunk 2]
```diff
@@ -1,92 +1,35 @@
...
-
-    ![image](https://github.com/mattermost/mattermost-plugin-confluence/assets/74422101/2ca5cc02-8a98-462d-b5f8-1cb20b7272d6)
-
-5. In **From this URL**, enter the URL that is displayed after you typed `/confluence install cloud` - it is unique to your server. You'll need to copy/paste it from Mattermost into the **From this URL** field in Confluence. 
-6. Once installed, you will see the "Installed and ready to go!" message in Confluence.
-7. You can now go to Mattermost and type `/confluence subscribe` in the channel you want to get notified by Confluence.
-
 ## Configure notifications
 
 ### /confluence subscribe
```
Comment (mickmister) [inline README.md:35]: Not sure if this is an admin-only feature, but either way it seems like it's the only feature so it makes sense to be separate from the admin guide :+1:

**Review by mickmister — APPROVED** [review]
Thanks for the contribution @imkrishnasarathi! :tada:

The changes LGTM. I also have one further suggestion to improve the README while we are visiting this documentation

File: README.md (hunk 1 above)
Comment (imkrishnasarathi) [reply README.md]: done.

File: README.md (hunk 2 above)
Comment (imkrishnasarathi) [reply README.md]: done

File: docs/admin-guide.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+### Prerequisites
+
+For Confluence Plugin, Mattermost server v5.19+ is required. Confluence Server version 7.x+ is supported. Confluence Data Center is not certified.
+
+### Installation
+
+To get started, a plugin that is installed on the Mattermost Server is needed to receive messages from the Confluence server. The notifications route it to the correct channel based on subscriptions that are configured. 
+
+#### Marketplace installation (Recommended)
```
[the remark this answers is not in the document]
Comment (imkrishnasarathi) [reply docs/admin-guide.md]: done.

**Review by mickmister — APPROVED** [review]

**Comment by imkrishnasarathi** [comment]
@cwarnermm I request you to check if it's okay now.

**Comment by imkrishnasarathi** [comment]
Hello team mattermoest! Updates?

**Comment by hanzei** [comment]
@cwarnermm Heads up that the PR is ready for a final a review.

**Review by cwarnermm — APPROVED** [review]
Thank you, @imkrishnasarathi!

**Pull request merged**